### PR TITLE
[9.x] Fix SES V2 Transport "reply to" addresses

### LINKED ISSUE #47512: SES v2 Transport - ReplyTo address incorrectly configured
### Laravel Version

9.52.9 - All versions will be impacted

### PHP Version

8.0

### Database Driver & Version

_No response_

### Description

SESv2 mail transport is incorrectly configuring the reply to address to be the email from address.
This seems to have come about as the SES v2 API method that's now being used is called `sendEmail` and I suspect when this new transport was created a number of items were copied over from the original transport class without updating them.

This is affecting all Versions of Laravel and anyone using the SES v2 transport.

### Steps To Reproduce

Create and send an email via the ses-v2 transport.
The received email will have the optional reply-to address set to the sender address.

If a reply-to the address has been specified it will be ignored.

### PR DESCRIPTION
**Pull request by jacobmllr95**
When I added the `SesV2Transport` a few months back, I used the `ReplyToAddresses` attribute to define the sender of the message. This always set the sender as "reply to" address and prevented user-defined addresses to be applied.

Closes #47512.

### CODE REVIEW

**Review by driesvints — APPROVED** [review]

**Comment by ArthurGuy** [comment]
How often are new version tags created? i.e. when will this fix be available for use

**Comment by GrahamCampbell** [comment]
This fix is already immediately available using dev stability.

**Comment by ArthurGuy** [comment]
This is for a service running a production workload so I need to wait for a tagged release.

**Comment by GrahamCampbell** [comment]
There is no reason to wait for a tagged release. The lock file allows you to be pinned to a specific commit hash.

**Comment by ArthurGuy** [comment]
That would require the team always being aware when any other composer update is made so dev changes aren't inadvertently pulled down, and also keeping track of this issue to roll back the version being pinned when it does eventually get tagged.

We are running a product in a regulated space where a lot is at stake, I am sure others might make a different call but it's just not worth the risk to me.

**Comment by ArthurGuy** [comment]
Thanks @mfn I didn't realise you could do that, I assumed it was the hash in the lock file that was being referred to.
I will leave it to Monday, just in case, but thats definitely worth exploring as an option.

**Pull request merged**
